automation/taskcluster/androidTest/ui-test.sh

```diff
@@ -95,30 +127,5 @@ echo
 $JAVA_BIN -jar $FLANK_BIN android run --config=$flank_template --max-test-shards=$num_shards --app=$APK_APP --test=$APK_TEST --project=$GOOGLE_PROJECT
 exitcode=$?
 
-echo
-echo
-echo "COPY ARTIFACTS"
-echo
-cp -r "./results" "./test_artifacts"
-exitcode=$?
 failure_check
```

Comment:
good!  making this consistent w/ Fenix.  Let me know if you think these or more or less identical or if we could make this generic enough to share across fenix, a-c, r-b...